src/web/components/LabsHeader.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { from } from '@guardian/src-foundations/mq';
+import { border, labs, space } from '@guardian/src-foundations';
+
+import { Section } from '@frontend/web/components/Section';
+import { Dropdown } from '@root/src/web/components/Dropdown';
+
+import LabsLogo from '@frontend/static/logos/the-guardian-labs.svg';
+import { textSans } from '@guardian/src-foundations/typography';
+import { LinkButton } from '@guardian/src-button';
+import { SvgArrowRightStraight } from '@guardian/src-icons';
+import { Link } from '@guardian/src-link';
+
+const Container = ({ children }: { children: React.ReactNode }) => (
+	<div
+		className={css`
+			position: relative;
+			height: 55px;
+			display: flex;
+			justify-content: space-between;
+		`}
+	>
+		{children}
+	</div>
+);
+
+const Left = ({ children }: { children: React.ReactNode }) => (
+	<div
+		className={css`
+			display: flex;
+		`}
+	>
+		{children}
+	</div>
+);
+
+const Right = ({ children }: { children: React.ReactNode }) => (
+	<div
+		className={css`
+			display: flex;
+			padding: ${space[1]}px 0;
+		`}
+	>
+		{children}
+	</div>
+);
```

Comment:
I have mixed feeling about these components, could we not just use thier css values to define them? or at least rename them to be a little more decriptive than Left and Right (maybe something like LeftCol?)

Reply:
LeftCol already has a strong meaning as a breakpoint. I've used Left and Right in a few places to denote content that is moved to the left and right in a few places. There's also Top and Bottom. I find it helps when reading the layout code to know where things will appear 🤷